php/class-plugin.php

```diff
@@ -291,6 +291,8 @@ public function process_post_query_vars( $post_query_vars ) {
 			'order',
 		);
 
+		$allowed_query_vars = apply_filters( 'customize_object_selector_query_posts_allowed_vars', $allowed_query_vars );
```

Comment:
@miina there's no need for this `customize_object_selector_query_posts_allowed_vars` filter now because the opt-in will instead be accounted for below in the `customize_object_selector_post_query_vars` filter.